packages/components/src/components/quickfact/quickfact.ts

```diff
@@ -0,0 +1,154 @@
+import '../icon/icon';
+import { css, html } from 'lit';
+import { customElement } from '../../internal/register-custom-element';
+import { property } from 'lit/decorators.js';
+import { setDefaultAnimation } from '../../utilities/animation-registry';
+import cx from 'classix';
+import SdAccordion from '../accordion/accordion';
+
+/**
+ * @summary Accordion shows a brief summary and expands to show additional content.
+ * @documentation https://solid.union-investment.com/[storybook-link]/quickfact
+ * @status stable
+ * @since 3.9.0
+ *
+ * @slot icon - Optional icon to show in the header. Works best with `<sd-icon>`.
+ *
+ * @csspart icon - The container that wraps the icon.
+ *
+ * @dependency sd-icon sd-accordion
+ */
+@customElement('sd-quickfact')
+export default class SdQuickfact extends SdAccordion {
+  /**
+   * Determines if the quickfact is not interactive. When set to `true`, the quickfact will not expand or collapse.
+   */
+  @property({ type: Boolean, reflect: true, attribute: 'not-interactive' }) notInteractive = false;
+
+  /** @internal */
+  RenderSummary = () => {
+    return html`
+      <slot
+        name="summary"
+        part="summary"
+        class=${cx('flex flex-auto items-center text-left', this.notInteractive ? 'text-black' : 'text-primary')}
+        >${this.summary}</slot
+      >
+    `;
+  };
+
+  /** @internal */
+  RenderSummaryIcons = () => {
+    return html` <span
+      part="summary-icon"
+      class=${cx(
+        'flex flex-grow-0 flex-shrink-0 flex-auto items-center transition-all ease-in-out duration-300 text-xl',
+        this.open && 'rotate-180',
+        this.notInteractive && 'hidden'
+      )}
+      ><slot name="expand-icon" class=${cx(this.open && 'hidden')}>
+        <sd-icon library="system" name="chevron-down"></sd-icon>
+      </slot>
+      <slot name="collapse-icon" class=${cx(!this.open && 'hidden')}>
+        <sd-icon library="system" name="chevron-down"></sd-icon> </slot
+    ></span>`;
+  };
+
+  /** @internal */
+  RenderDefaultSlot = () => {
+    return html` <div part="content" id="content" class=${cx('overflow-hidden', this.notInteractive && 'hidden')}>
+      <slot part="content__slot" class="block px-4 py-6" role="region" aria-labelledby="header"></slot>
+    </div>`;
+  };
+
+  /** @internal */
+  RenderQuickfactIcon = () => {
+    return html` <div part="icon" class="flex flex-grow-0 flex-shrink-0 flex-auto items-center text-xl">
+      <slot name="icon"></slot>
+    </div>`;
+  };
```

Comment:
We also have these three render functions in the Accordion. Wouldn't a service be better here?